### ISSUE TITLE
[BUG] webstart.jnlp file is empty after altering server context path

### ISSUE THREAD

**Issue by fridlik** [nextgenhealthcare/connect#4462]
**Describe the bug**
I am not able to connect to mirth server running behind reverse proxy from the Administrator Launcher, when the mirth is configured with non default context path (e.g. /mirth).

**To Reproduce**
1. Install the Server and Administrator
2. change contextpath in `mirth.properties` to `http.contextpath = /mirth/`
3. try to connect using Administrator Launcher using http://127.0.0.1:8080/mirth/
4. You can also try to setup reverse proxy in front of the mirth server (I tried Apache2) with the same result:
    ```
    <Location /mirth/>
        ProxyPass http://127.0.0.1:8080/
        ProxyPassReverse http://127.0.0.1:8080/
    </Location>
    ``` 

**Expected behavior**
The Administrator Launcher is able to connect.

**Actual behavior**
The Administrator Launcher is not able to connect and returns the following error: 
```The markup in the document preceding the root element must be well-formed.
org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 3; The markup in the document preceding the root element must be well-formed.
	at com.sun.org.apache.xerces.internal.parsers.DOMParser.parse(DOMParser.java:257)
	at com.sun.org.apache.xerces.internal.jaxp.DocumentBuilderImpl.parse(DocumentBuilderImpl.java:339)
	at javax.xml.parsers.DocumentBuilder.parse(DocumentBuilder.java:121)
	at com.mirth.connect.client.launcher.f.a(SourceFile:673)
	at com.mirth.connect.client.launcher.f.a(SourceFile:167)
	at com.mirth.connect.client.launcher.MirthClientLauncher.run(SourceFile:1164)
	at java.lang.Thread.run(Thread.java:748)
```

**Screenshots**
[
<img width="1076" alt="Screenshot 2021-02-22 at 18 06 49" src="https://user-images.githubusercontent.com/1500518/108743305-91cddd80-7530-11eb-90e4-94ea1f852f65.png">
](URL)

**Environment (please complete the following information):**
 - OS server: tested on Centos 7 from rpm package and in docker image nextgenhealthcare/connect:3.10.1
 - Java: tested with java 8 and java 11
 - Connect Version 3.10.1

**Workaround(s)**
I did not find any workaround.

**Additional context**
I have found an additional issue when testing this one. The server returns the wrong http header `X-Content-Type-Options?: nosniff`, the question mark cannot be there. This can cause additional problems...

**Comment by cturczynskyj** [comment]
Looks like a bug! Specifically, when altering the http.contextpath in mirth.properties, the webstart.jnlp file turns up empty. Created an internal issue and will be looking into it. Thanks!

**Comment by MichaelLeeHobbs** [comment]
Dup #4432 not a complete dup.

**Comment by nicholass-alcidion** [comment]
Is there any chance of getting the fix for this issue into a bug-fix release for 3.10.x or will we just need to wait for 3.11 for the resolution?
